### DISCUSSION TITLE
Esmerald vs Litestar?

### DISCUSSION THREAD

**Discussion by pikhovkin** [dymmond/esmerald#235] [General]
What is Esmerald better than [Litestar](https://github.com/litestar-org/litestar)?

**Comment by tarsil** [comment]
@pikhovkin i think Litestar is a fantastic framework that serves its own use cases. 

Esmerald is built in the top of Starlette and focused on removing boilerplating in terms of coding. 

Esmerald also adopted a friendly syntax for those familiar with other frameworks like Django for example and the integration with DBs, ODM, brings a scheduler if you want to.... In the end it's up to you to choose which one serves you better. 

Esmerald vs Litestar is never a competition at all.

**Comment by PeterChain** [comment]
@pikhovkin Litestar and Esmerald are two different things with different objectives.
Esmerald is designed more to complete applications, not only API.

The goal here is simplicity for those who are not from the python world while leveraging new features (eg: async).
